src/DsaCommon.hpp

```diff
@@ -12,7 +12,11 @@
 #ifndef _BASETSD_H_
 typedef unsigned char  BYTE;
 typedef unsigned short WORD;
+#ifdef _WIN32
 typedef unsigned long  DWORD;

```

Comment:
On Windows 64-bit `long` will have 64-bit length, is this correct?

Reply:
They are fixed to 32 on both, it's specific of windows.
Backwards compatibility you know, crappy code that has assumed 32 bit sizes.

Reply:
Then, fine, yeah, because of that damn, Windows sucks, but, I am glad because `stdint.h` exists to solve this crap! :fox_face: